coralproject/cay
# fix multiple choice title blanking out

# NOT READY - REFACTOR UNDERWAY
## What does this PR do?

fixes: https://github.com/coralproject/ask/issues/94

The multiple choice (and probably other types) were being blanked out whenever the component received new props -- which could be anytime. I removed the lifecycle function that caused this misbehavior.

**WARNING** I'm not positive if this will have any side effects. please advice @pablocubico 
## How do I test this PR?
- Create a new Form
- add a multiple choice entry
- add a title
- click like mad around in the multiple choice component, and it should not clear out your title

@coralproject/frontend
